utils/db.ts

```diff
@@ -37,7 +43,12 @@ export async function createItem(initItem: InitItem) {
 export async function getAllItems(options?: Deno.KvListOptions) {
   const iter = await kv.list<Item>({ prefix: ["items"] }, options);
   const items = [];
-  for await (const res of iter) items.push(res.value);
+  for await (const res of iter) {
+    const { id } = res.value;
+    const commentsCount = await getItemCommentsCount(id);
+    res.value = { commentsCount, ...res.value };
+    items.push(res.value);
+  }
   return items;
 }
```

Comment:
I prefer to leave this function alone and instead map up comment counts within the `GET /` handler. You can see how I did this in #144.